src/apps/uclient/startuclient.c

```diff
@@ -1038,16 +1038,15 @@ int start_connection(uint16_t clnet_remote_port0, const char *remote_address0, c
         }
         ioa_addr arbaddr;
         make_ioa_addr((const uint8_t *)sarbaddr, 333, &arbaddr);
-        int i;
-        int maxi = (unsigned short)turn_random() % EXTRA_CREATE_PERMS;
-        for (i = 0; i < maxi; i++) {
+        const int maxi = (unsigned short)turn_random() % EXTRA_CREATE_PERMS;
+        for (int i = 0; i < maxi; i++) {
           uint16_t chni = 0;
           int port = (unsigned short)turn_random();
           if (port < 1024) {
             port += 1024;
           }
           addr_set_port(&arbaddr, port);
-          uint8_t *u = (uint8_t *)&(arbaddr.s4.sin_addr);
+          uint8_t *const u = (uint8_t *)&(arbaddr.s4.sin_addr);
```

Comment:
my brain refuses to compile the next line...
pointer is const - I get it. But writing to `u` defeats the purpose
Yes, content of u is not const
But still - what does const add here?

Reply:
I believe this is addressed now